src/common/components/Recaptcha/RecaptchaWrapper.tsx

```diff
@@ -21,6 +26,17 @@ interface RecaptchaWrapperProps {
   $log: any
 }
 
+export const findExistingScript = (): boolean => {
+  const scripts = document.getElementsByTagName('script')
+
+  for (let script of scripts) {
+    if (script.src.startsWith('https://www.google.com/recaptcha/api.js?render=')) {
```

Comment:
May be worth doing a more general lookup to catch things like `//www.google.com/recaptcha/api.js?render=`, but this is working on all the current implementations.